demos/keypad_encoder/lv_demo_keypad_encoder.c

```diff
@@ -166,9 +166,6 @@ static void msgbox_create(void)
     lv_obj_add_event_cb(mbox, msgbox_event_cb, LV_EVENT_ALL, NULL);
     lv_group_focus_obj(lv_msgbox_get_btns(mbox));
     lv_obj_add_state(lv_msgbox_get_btns(mbox), LV_STATE_FOCUS_KEY);
-#if LV_EX_MOUSEWHEEL
-    lv_group_set_editing(g, true);
-#endif
```

Comment:
Why is it removed?

Reply:
Because `LV_EX_MOUSEWHEEL` has not been defined in `lv_conf.h`.

Reply:
Do you think it is better to set the `editing` of the `group` here to on or off by default?

Reply:
> Because LV_EX_MOUSEWHEEL has not been defined in lv_conf.h.

Ahh, it worked when the examples were in `lv_examples.h`. For the sake of simplicity let's remove it.